src/Manifest/ManifestReader.php

```diff
@@ -119,13 +133,13 @@ protected function resolveManifestVersion(ModuleTransfer $moduleTransfer, string
             return $filePath;
         }
 
-        $nextSuitableVersion = $this->findNextSuitableVersion($moduleRecipiesDir, $moduleVersion);
+        $nextSuitableVersion = $this->findNextSuitableVersion($moduleRecipesDir, $moduleVersion);
```

Comment:
They are not called recipies anymore, but manifests
best to use this new name if you already change variables.